ext/civicrm_admin_ui/ang/afformTabMember.aff.json

```diff
@@ -0,0 +1,32 @@
+{
+    "type": "search",
+    "requires": null,
+    "entity_type": null,
+    "join_entity": null,
+    "title": "Memberships",
+    "description": null,
+    "placement": [
+        "contact_summary_tab"
+    ],
+    "summary_contact_type": [
+        "Organization"
+    ],
+    "summary_weight": 30,
+    "icon": "fa-id-badge",
+    "server_route": "",
+    "is_public": false,
+    "permission": [
+        "access CiviCRM",
+        "access CiviMember"
+    ],
+    "permission_operator": "AND",
+    "redirect": null,
+    "submit_enabled": true,
+    "submit_limit": null,
+    "create_submission": false,
+    "manual_processing": false,
+    "allow_verification_by_email": false,
+    "email_confirmation_template_id": null,
+    "navigation": null,
+    "modified_date": "2024-01-09 13:48:38"
```

Comment:
@jitendrapurohit our new convention is to use `.aff.php` files instead of `.aff.json` (the php format allows the use of `E::ts()` for the title). And "modified date" shouldn't be included.
If you don't want to convert these files by hand, you can use the command `civix export Afform afformTabMember` (that command will generate the `aff.html` and `.aff.php` files plus all the `.mgd.php` for embedded search displays in the form).

Reply:
yes, thanks for the above. Have updated 👍